sphinxmermaid/__init__.py

```diff
@@ -12,7 +13,7 @@
 
 LOGGER = logging.getLogger(__name__)
 MERMAID_JS_URL = "https://unpkg.com/mermaid/dist/mermaid.min.js"
-JS_INIT_MERMAID = "mermaid.initialize({startOnLoad:true});"
+DEFAULT_MERMAID_INIT = "mermaid.initialize({startOnLoad:true});"
```

Comment:
This may be defined as the default parameters dictionary, and not as a raw javascript instruction, don't you think ?

It could then become:

```python
DEFAULT_MERMAID_INIT = {"startOnLoad": True}

# ...

def create_mermaid_init(app: "Sphinx"):
    mermaid_init = app.config.mermaid_init
    if mermaid_init:
        check_mermaid_init(mermaid_init)
    else:
        mermaid_init = {}
    mermaid_init.update(DEFAULT_MERMAID_INIT)
    params = json.dumps(mermaid_init)
    return f"mermaid.initialize({params})"
```

Reply:
I agree. See my commit below for the update.

Instead of updating the user-specified mermaid_init with the default values, I believe that the user should be able to completely override the default dictionary. In my version, the default is only used if `sphinxmermaid_mermaid_init` is not set.

Reply:
All right, that's fine with me.